### PR TITLE
Azureiothubclient remove

### PR DESCRIPTION
**Pull request by GregTerrell**
Removed AzureIoTHubClient.cpp (.h); 

Updated example (http) by removing AzureIoTHubClient references in .ino: instantiation of iotHubClient object, iotHubClient.begin() in setup() and iotHubClient.setEpochTime() in initTime()

This is my 1st pull request, hope I did it correctly mamokarz.  I will wait for results before doing pull-requests for the other 3 project examples (http, mqtt, utility).  Thanks: GT

### CODE REVIEW

**Comment by BertKleewein** [comment]
Hi @GregTerrell, I'll be getting  this code checked in, but the process is slightly more involved than accepting a pull request -- it has to go into a different repo and then get moved over here via some method that I'm just learning about.  Don't worry, I'll do all the work and I'll keep you informed of my progress.

File: examples/simplesample_http/simplesample_http.ino [hunk 1]
```diff
@@ -130,8 +117,6 @@ void initTime() {
     Serial.print("Fetched NTP epoch time is: ");
     Serial.println(epochTime);
 
-    iotHubClient.setEpochTime(epochTime);
```
Comment (BertKleewein) [inline examples/simplesample_http/simplesample_http.ino]: epochTime is now an unused variable.  Is there any reason to retrieve this from the NTP server if we're not using this?

**Comment by GregTerrell** [comment]
I don't think there is a need to keep epochTime, I guess I assumed the IotHub client was referencing the device time somewhere further in the initialization.  We would keep it for some cases, but that is application code issue not library.

**Comment by GregTerrell** [comment]
I thought I deleted that line, sorry that should go.  I am still working our my github workflows.

**Comment by BertKleewein** [comment]
Code was manually merged. I did this by copying to this change to https://github.com/Azure/azure-iot-sdk-c and then using our build scripts to propagate the changes back to this repo.

**Comment by GregTerrell** [comment]
Thanks, there is the same issue ( iotHubClient ) in the http, mqtt, and utility repos (they all have a simpleSample example).  Would you like me to do (attempt... still learning) a pull-request to facilitate getting those in-sync?  

I know when I see conflicting structures in examples I am often left confused on which is correct.

**Comment by BertKleewein** [comment]
Don't be worried, these are unusual projects and they don't follow the usual rules or processes. I think pull requests might be the best way right now.

Thanks for keeping an eye on consistency. It really does make a difference.

_____________________________

**Pull request closed without merging**